gradle.properties

```diff
@@ -5,20 +5,21 @@
 group=org.jetbrains.kotlinx
 version=0.20.0-SNAPSHOT
 
-kotlin.version=1.4-M2-eap-62
+kotlin.version=1.4-M2-eap-72
 
 # This version take precedence if 'bootstrap' property passed to project
 kotlin.version.snapshot=1.4-SNAPSHOT
 # Also set KONAN_LOCAL_DIST enviroment variable in bootstrap mode to auto-assign konan.home
 
 junit_version=4.12
 infra_version=0.1.0-dev-47
-bintray_version=1.8.4-jetbrains-5
-benchmarks_version=0.1.7-dev-14
 jackson_version=2.10.0.pr1
 dokka_version=0.10.1
 native.deploy=false
 
+kotlin.mpp.enableGranularSourceSetsMetadata=true
+kotlin.mpp.enableCompatibilityMetadataVariant=true
```

Comment:
What does this flag do?

Reply:
`enableCompatibilityMetadataVariant` is a special mode for libraries published with HMPP that adds an old-style `*.kotlin_metadata` variant that can be consumed by non-HMPP users. It is only useful for migration period until we enable HMPP by default.

Reply:
In HMPP we've changed binary format of artifact published from common code. Previously it was a `.jar`-file with `.kotlin_metadata` inside, now it's a `.klib` without IR (well, essentially it's still an archive with `.kotlin_metadata` inside, but with different extension and layout)

This means that non-HMPP clients won't be able to consume that artifact. At the moment,  HMPP is not enabled by default, hence we need to publish a compatibility variant with old-style metadata .jar for anyone who hasn't enabled HMPP manually. 

If we enable HMPP by default in 1.4, then, most probably, we'll be able to drop this legacy-variant.